sdk/Trace/TraceContextPropagator.php

```diff
@@ -16,40 +22,55 @@
  * traceparent header and relevant parts of the tracestate header containing
  * their proprietary information.
  */
-final class TraceContext implements API\TextMapFormatPropagator
+final class TraceContextPropagator implements API\TextMapPropagator
```

Comment:
Renamed this to make it more clear what it is, and updated implementation to better follow the new interface. Used https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/trace/propagation/W3CTraceContextPropagator.java as a reference.